active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,591 @@
+
+<a href="url"><img src="TeachingTipLightDismissSample.jpg" height="600"></a>
+
+
+## Create a teaching tip
+
+<!-- Use the Password property to get or set the contents of the PasswordBox. You can do this in the handler for the PasswordChanged event to perform validation while the user enters the password. Or, you can use another event, like a button Click, to perform validation after the user completes the text entry. -->
+
+Here's the XAML for a teaching tip control that demonstrates the default look of the TeachingTip with a title and subtitle. 
+
+When the user clicks the button, a teaching tip will appear and display a message to the user. 
+
+XAML
+```XAML
+<Button Content="Save" Click="OnFirstSaveButtonClick">
+    <muxc:TeachingTip.Attach> 
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to.">
+        </muxc:TeachingTip>
+    </muxc:TeachingTip.Attach>
+</Button>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+   AutoSaveTip.IsOpen = true;
+}
+```
+
+Here's the result when this code runs and the clicks the button.
+
+<a href="url"><img src="TeachingTipSampleApp.jpg" height="600"></a>
+
+### Add a margin 
+
+A teaching tip can have a margin added by setting the Margin property. When a teaching tip is attached to a UI element, a margin will move it away from the UI element it is pointing to. When a teaching tip is not attached to anything, a margin will move it away from the side of the app window it is positioned towards. 
+
+XAML
+```XAML
+<Button Content="Save" Click="OnFirstSaveButtonClick">
+    <muxc:TeachingTip> 
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to."
+            Margin="80">
+        </muxc:TeachingTip>
+    </muxc:TeachingTip.Attach>
+</Button>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+   AutoSaveTip.IsOpen = true;
+}
+```
+
+<a href="url"><img src="TeachingTipMarginSample.jpg" height="600"></a>
+
+### Point to a UI element
+
+You can change the teaching tip to point to a specific UI element by setting the TeachingTipPlacementMode property. For a targeted teaching tip, this placement will be relative to the UI element target. For a non-targeted teaching tip, this placement will relative to app window.  
+
+XAML
+```XAML
+<Button Content="Tip Example" Click="OnFirstSaveButtonClick">
+    <muxc:TeachingTip.Attach> 
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to."
+            Margin="15"/>
```

Comment:
```suggestion
            Margin="12"/>
```
This is extremely minor, but our designers work on a 4px visual grid, so using sizes and margin values that are divisible by 4 better fits with their thinking. If you were to look through app redlines, the most common margin sizes will be 12px (for small windows/screens) and 24px (for medium and large windows/screens). So using those values feels more on brand :)

Reply:
Thank you! These pictures were artificially constructed while the implementation is being developed. This led to me having to take some liberty with the art due to the shadow and the complication it presents to merging into photo edits with integrity. When these photos are updated with real examples, I intend to have the Margin example being the only one making use of the margin property.